NEWS.rst

```diff
@@ -4,6 +4,10 @@ In development
 API changes
 -----------
 
+``/service/ou/create``:
+
+If an empty user_key is supplied, the uuid of the orgunit is used
```

Comment:
This is no longer true, is it?

Reply:
No - it will be if NO userkey - will fix it

Reply:
I removed - it is now standard, isn't it